tests/hierarchical/hierarchical_non_uniform_local_range.cpp

```diff
@@ -50,6 +50,17 @@ void check_expected(const std::vector<sycl::int3> &data, unsigned local_id,
   }
 }
 
+// there is no id -> range conversion in the spec
+static sycl::range<1> id_to_range(sycl::id<1> id) {
+  return sycl::range<1>(id[0]);
+}
+static sycl::range<2> id_to_range(sycl::id<2> id) {
```

Comment:
```suggestion
static sycl::range<2> to_range(sycl::id<2> id) {
```
since the `id` is clear as the parameter?

Reply:
This is obsolete now because it has been fixed on SYCL-2020 branch in the meantime.